models/repo.go

```diff
@@ -2528,6 +2569,12 @@ func CheckRepoStats() {
 		log.Error("Select %s: %v", desc, err)
 	} else {
 		for _, result := range results {
+			select {
+			case <-ctx.Done():
```

Comment:
Can this be moved to ctx?:
```
if ctx.ShutdownCheck() {
    return
}
```
On the other hand, `Aborting due to shutdown` is not very descriptive because it doesn't say _what_ was aborted.

Maybe:
```
if ctx.ShutdownCheck("CheckRepoStats()") {
    return
}
```

Reply:
Nope. Can't change the context interface - it's part of core Go.

Reply:
I've added CheckRepoStats to the message.